src/infrastructure/validation/logger/ILogger.java

```diff
@@ -1,17 +1,18 @@
-/**
- * 
- */
 package infrastructure.validation.logger;
 
 /**
+ * Interface for interacting with the Logger instance
  * @author Navaneeth M Nambiar
  */
 public interface ILogger {
+	
 	/**
+	 * log method will be be used to specify to the Logger instance to log, 
+	 * the particular event
 	 * 
-	 * @param moduleIdentifier
-	 * @param level
-	 * @param message
+	 * @param moduleIdentifier the module which is calling the logger. @see logger.ModuleID
+	 * @param level            the level of the log to be used. @see logger.LogLevel
+	 * @param message          message to be printed in the log
```

Comment:
Vertical alignment between different @param descriptions is not compulsory

Reply:
yeah but the current one is more readable. So, I'll keep it as is

Reply:
Thats fine